src/libraries/System.Runtime/tests/System.Diagnostics.Debug.Tests/System.Diagnostics.Debug.Tests.csproj

```diff
@@ -13,6 +13,7 @@
     <ProjectReference Include="$(CoreLibProject)" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime\src\System.Runtime.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Collections\src\System.Collections.csproj" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Memory\src\System.Memory.csproj" />
```

Comment:
Without this the build would fail with:

> error CS0656: Missing compiler required member 'System.MemoryExtensions.AsSpan'

That's because that helper is now used for codegen of the built-in span conversions from string to ReadOnlySpan<char>. (Previously the user-defined conversion on `string` was used, but that's not available on .NET Framework.)

Reply:
I'm not understanding this one. For .NET Framework, Span itself is part of the System.Memory package. How could you be in a situation where you're using Span on netfx but don't have the System.Memory reference in your transitive closure?

Reply:
Maybe it's just some weirdness of this project. It has a reference to Span via `\src\libraries\System.Runtime\ref\System.Runtime.cs`. That contains definition of Span but not MemoryExtensions.